connector/src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala

```diff
@@ -130,13 +127,14 @@ private[bigquery] class DirectBigQueryRelation(
         val session = client.createReadSession(
           CreateReadSessionRequest.newBuilder()
             .setParent(s"projects/${options.parentProject}")
-            .setFormat(readDataFormat)
-            .setRequestedStreams(maxNumPartitionsRequested)
-            .setReadOptions(readOptions)
-            .setTableReference(actualTableReference)
+            .setReadSession(ReadSession.newBuilder()
+              .setDataFormat(readDataFormat)
+              .setReadOptions(readOptions)
+              .setTable(actualTablePath
+            ))
+            .setMaxStreamCount(maxNumPartitionsRequested)
             // The BALANCED sharding strategy causes the server to assign roughly the same
```

Comment:
Sharding Strategy is not available in v1 API. Is there any other known way to achieve this functionality? @davidrabinowitz

Reply:
The v1 API has removed the sharding strategy.